config/src/main/java/com/alibaba/nacos/config/server/controller/HistoryController.java

```diff
@@ -143,5 +147,19 @@ public ConfigHistoryInfo getPreviousConfigHistoryInfo(@RequestParam("dataId") St
         checkHistoryInfoPermission(configHistoryInfo, dataId, group, tenant);
         return configHistoryInfo;
     }
+
+    /**
+     * Query configs list by namespace.
+     * @param tenant config_info namespace
+     * @since 2.1.1
+     * @return list
+     */
+    @GetMapping(value = "/configs")
+    public List<ConfigInfoWrapper> getDataIds(@RequestParam("tenant") String tenant) {
```

Comment:
add secure annotation to add auth check.

Reply:
> I have a question about this feature.
> 
> According to previous feedback, users prefer to use the fuzzy search function, and this PR will cause the fuzzy search function to be abandoned. So I tend to keep input not select.
this select is  have fuzzy search function

Reply:
According to different inputs, the corresponding matching data can be fuzzy searched

Reply:
Please Add the Secure annotation to add auth check.